tools/ci/README.md

```diff
@@ -24,3 +24,42 @@ with the chosen name:
 ```bash
 docker run -t <my-custom-name>
 ```
+
+# Ydoc NodeJS Docker
+
+## Building
+
+To build a NodeJS-based Ydoc, you need to first ensure that you have the
+distributable sources:
+
+```bash
+pnpm -r compile
+```
+
+the resulting artifacts are located in `app/ydoc-server-nodejs/dist` directory.
+Having the right NodeJS sources in place, one can now build the docker image:
+
+```bash
+ocker build -t <docker-image-name> -f tools/ci/docker/ydoc-server/Dockerfile --build-context docker-tools=tools/ci/docker/ydoc-server app/ydoc-server-nodejs
+```
+
+## Running
+
+One should always start Ydoc with the right configuration:
+
+- PORT - the port number under which Ydoc will be available
+- HOSTNAME - the hostname under which Ydoc will be available
+- LANGUAGE_SERVER_URL - the full url (with port number) of the language server
+  to connect to
+
+```bash
+docker run -t <docker-image-name> -e PORT=1234 -e HOSTNAME='0.0.0.0' -e LANGUAGE_SERVER_URL=ws://localhost:59876 ydoc-server-nodejs:latest
```

Comment:
Is the line correct? I suppose the `-t` flag just allocates TTY, and the docker image name is supplied as the last argument `ydoc-server-nodejs:latest`
```suggestion
docker run -t -e PORT=1234 -e HOSTNAME='0.0.0.0' -e LANGUAGE_SERVER_URL=ws://localhost:59876 ydoc-server-nodejs:latest
```